core/kazoo_attachments/src/kz_att_azure.erl

```diff
@@ -49,38 +97,3 @@ azure_pid(Account, Key) ->
             end;
         Pid -> Pid
     end.
-
--spec put_attachment(kz_data:connection(), kz_term:ne_binary(), kz_term:ne_binary(), kz_term:ne_binary(), kz_term:ne_binary(), kz_data:options()) -> any().
-put_attachment(Settings, DbName, DocId, AName, Contents, _Options) ->
-    CT = kz_mime:from_filename(AName),
-
-    {Container, Name} = resolve_path(Settings, {DbName, DocId, AName}),
-    Pid = azure_pid(Settings),
-    AzureOptions = [{content_type, kz_term:to_list(CT)}, return_headers],
-    case erlazure:put_block_blob(Pid, Container, Name, Contents, AzureOptions) of
-        {ok, Headers, _Body} ->
-            props:to_log(Headers, <<"AZURE HEADERS">>),
-            Data = base64:encode(term_to_binary({Settings#{file => Name}, Name})),
-            Metadata = kz_json:from_list(kz_att_util:headers_as_binaries(Headers)),
-            {'ok', [{'attachment', [{<<"azure">>, Data}
-                                   ,{<<"metadata">>, Metadata}
-                                   ]}
-                   ]};
-        {error, _E} = Error ->
-            lager:error("error storing attachment in azure : ~p", [_E]),
-            Error
-    end.
-
-
--spec fetch_attachment(kz_data:connection(), kz_term:ne_binary(), kz_term:ne_binary(), kz_term:ne_binary()) ->
-                              {'ok', iodata()} |
-                              {'error', 'invalid_data' | 'not_found'}.
-fetch_attachment(HandlerProps, DbName, DocId, AName) ->
-    case kz_json:get_value(<<"azure">>, HandlerProps) of
-        'undefined' -> {'error', 'invalid_data'};
-        AzureData ->
-            {Settings, _ContentId} = binary_to_term(base64:decode(AzureData)),
-            Pid = azure_pid(Settings),
-            {Container, Name} = resolve_path(Settings, {DbName, DocId, AName}),
-            erlazure:get_blob(Pid, Container, Name)
-    end.
```

Comment:
this is very to check what the changes real are (if any). try not to move the code around.